backend/migrations/versions/799a2343955e_fix_moduletype_enum.py

```
@@ -0,0 +1,118 @@
+        "recognition_guide": "RECOGNITION_GUIDE",
+        "quiz": "QUIZ",
+        "challenge": "CHALLENGE",
+        "solution_guide": "CHALLENGE_SOLUTION",
+        "bonus_challenge": "BONUS_CHALLENGE",
+        "bonus_solution_guide": "BONUS_SOLUTION",
+    }
+
+    # Update the temporary column with the new enum values
+    for old_value, new_value in mapping.items():
+        op.execute(
+            f"UPDATE modules SET module_type_temp = '{new_value}' WHERE module_type = '{old_value}'"
```

Ruff (0.8.2)

Possible SQL injection vector through string-based query construction

(S608)